src/content/toys.ts

```diff
@@ -9,34 +9,54 @@
  */
 export function jack_in() {
     // chinese characters - taken from the unicode charset
-    const chinese = "田由甲申甴电甶男甸甹町画甼甽甾甿畀畁畂畃畄畅畆畇畈畉畊畋界畍畎畏畐畑".split(
-        "",
-    )
+    const chinese = "田由甲申甴电甶男甸甹町画甼甽甾甿畀畁畂畃畄畅畆畇畈畉畊畋界畍畎畏畐畑".split("")
     const colour = "#0F0" // green text
-    rain(chinese, colour)
+    rain(makeBlock(), chinese, colour)
 }
 
-export function no_mouse() {
-    rain([" "], "#FFF", 0) // No characters, unused colour code, no darkening
+
+export function music() {
+    // music characters - taken from the unicode charset
+    const music = "𝄞𝄟𝄰𝅘𝅥𝅮𝅘𝅥𝅯𝅘𝅥𝅰𝄽".split("")
+    const colour = "#ead115"
+    rain(makeBlock(), music, colour)
 }
 
-export const snow = () => rain(["❄"], "#FFF", 0.15)
+export function no_mouse() {
+    makeBlock()
+}
 
-export function rain(characters: string[], colour, darkening = 0.05) {
+function makeBlock() {
     const d = document.createElement("div")
+    d.className = "_tridactyl_no_mouse_"
     d.style.position = "fixed"
     d.style.display = "block"
-    d.style.width = "100%"
-    d.style.height = "100%"
+    d.style.width = String(window.innerWidth)
+    d.style.height = String(window.innerHeight)
     d.style.top = "0"
     d.style.left = "0"
     d.style.right = "0"
     d.style.bottom = "0"
     d.style.zIndex = "1000"
     d.style.opacity = "0.5"
+    document.body.appendChild(d)
+    return d
+}
+
+export function removeBlock() {
+    Array.from(document.getElementsByClassName("_tridactyl_no_mouse_")).map(el=>{
+        if((el as any).intid!=null){
+            clearInterval((el as any).intid)
```

Comment:
`as any` is very lazy on my behalf. `eslint` will probably complain about it. I'm sure we can tighten it but I just wanted to get the code building for now

Reply:
Why is this necessary though, it should work otherwise too, no?

Reply:
We need to tell Typescript that `el` can have an `initd` property. Most HTML elements won't have one so without it you'll get a compiler error.

`eslint` doesn't like `any` types because it could literally be anything, even a function. You're basically turning off 
Typescript